layers/+lang/purescript/packages.el

```diff
@@ -71,17 +74,18 @@
       (customize-set-variable 'psc-ide-rebuild-on-save purescript-enable-rebuild-on-save)
 
       (add-to-list 'spacemacs-jump-handlers-purescript-mode 'psc-ide-goto-definition)
+      (spacemacs/declare-prefix "mmi" "imports")
       (spacemacs/set-leader-keys-for-major-mode 'purescript-mode
         "mt"  'psc-ide-add-clause
-        "mcs" 'psc-ide-case-split
+        "mc"  'psc-ide-case-split
```

Comment:
Make sure to update the purescript/README.org to keep it in sync with these changes :)

Reply:
Done. Also reorganized the table of keybindings so the nesting is a little more apparent.